langwatch/prisma/migrations/20260728120001_budgets_provider_filter_group_scope_routing_mode/migration.sql

```
@@ -0,0 +1,28 @@
+
+-- AlterTable
+ALTER TABLE "GatewayBudget" ADD COLUMN "providerKey" TEXT;
+
+-- AlterTable
+ALTER TABLE "GatewayBudgetLedger" ADD COLUMN "providerKey" TEXT;
+
+-- AlterTable
+ALTER TABLE "VirtualKey" ADD COLUMN "routingMode" "VirtualKeyRoutingMode" NOT NULL DEFAULT 'NONE';
+
+-- CreateIndex
+CREATE INDEX "GatewayBudget_scopeType_scopeId_providerKey_idx" ON "GatewayBudget"("scopeType", "scopeId", "providerKey");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)